docs/form-styling.rst

```diff
@@ -43,6 +45,20 @@ Adding an image to a question displays the image as part of the question.
   yesnodk, n, no
   yesnodk, dk, I don't know
 
+.. _big-image:
+
+Linking to big images for zooming and panning
```

Comment:
Title should be "Bigger images for panning and zooming".

Shorter title is nice because of the side bar. Most people say "panning and zooming". I checked with Google Trends.